### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by friedcucumber**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+## A. Header
+
+### Anatomy
+The header area displays a list header that you can click to opens detail list.
+
+<img width="50%" src="https://user-images.githubusercontent.com/12169834/223118601-0386f5c7-fd4c-4fe1-a03d-775b5e12a9e6.png"/>
+
+### Behavior
+
+Follow the steps below to open the full list of items on header click:
```
Comment (Abadzhev) [inline README.md:31]:
```suggestion
Follow the steps below to handle header clicks and implement navigation to the screen with the complete list of items:
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+
+1. Call the [Routing.RegisterRoute](https://learn.microsoft.com/en-us/dotnet/api/microsoft.maui.controls.routing.registerroute?view=net-maui-7.0#microsoft-maui-controls-routing-registerroute(system-string-system-type)) method to register the page that contains the full list of items:
```
Comment (Abadzhev) [inline README.md:33]:
```suggestion
1. Call the [Routing.RegisterRoute](https://learn.microsoft.com/en-us/dotnet/api/microsoft.maui.controls.routing.registerroute?view=net-maui-7.0#microsoft-maui-controls-routing-registerroute(system-string-system-type)) method to register the page that contains the complete list of items:
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+    The **NavigateToAll** method calls the [GoToAsync](https://learn.microsoft.com/en-us/dotnet/api/microsoft.maui.controls.shell.gotoasync?view=net-maui-7.0#microsoft-maui-controls-shell-gotoasync(microsoft-maui-controls-shellnavigationstate-system-collections-generic-idictionary((system-string-system-object)))) method to open the detail view.
+
+3. Specify the [QueryProperty](https://learn.microsoft.com/en-us/dotnet/api/microsoft.maui.controls.querypropertyattribute?view=net-maui-7.0) attribute for the **CompleteListViewModel** class to pass the clicked header context to the command registered in the previous step:
+   
+   ```csharp
+   namespace CollectionViewWithActionButtons.ViewModels {
+       [QueryProperty(nameof(ParentCard), "Parent")]
+       internal class CompleteListViewModel : INotifyPropertyChanged {
+           public Card parentCard;
+           public Card ParentCard {
```
Comment (Abadzhev) [inline README.md:88]: Why are both members public: "parentCard" and "ParentCard"? Isn't this a mistake?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,279 @@
...
+               PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
+           }
+       }
+   }
+   ```
+
+## B. List Items
+
+
+This area contains list of items that you can open (click on it) or remove (the **✕** button on the right).
```
Comment (Abadzhev) [inline README.md:106]:
```suggestion
This area contains a list of items. You can click on an item to view its details. Click on an **✕** button to delete the corresponding item. 
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,279 @@
...
+        <Grid ...>
+            <Image Source="{Binding Icon}" .../>
+            <Label Text="{Binding Name}" .../>
+        </Grid>
+    </dxco:SimpleButton>
+    <dxco:SimpleButton Text="&#x2715;" Command="{Binding Source= {RelativeSource Mode=FindAncestorBindingContext,
+                    AncestorType={x:Type viewModels:Card}}, Path=HideCommand}" CommandParameter="{Binding}" .../>
+    ```
+
+1. Define the **Card** class's **ItemClick** and **Hide** commands:
```
Comment (Abadzhev) [inline README.md:130]:
```suggestion
1. Define the **ItemClick** and **Hide** commands in the **Card** class:
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,279 @@
...
+
+## C. Footer (Optional)
+
+The footer area includes buttons that hide or apply all the items from this group. 
+
+<img width="50%" src="https://user-images.githubusercontent.com/12169834/223119520-25279691-b6cd-4147-b3aa-8154d0b9670e.png"/>
+
+### Behavior
+
+Buttons within the footer are visible only when the **Card** class's **AllowCommonActions** property is `true`. The **PrimaryActionName** and **SecondaryActionName** properties define button names.
```
Comment (Abadzhev) [inline README.md:184]:
```suggestion
Buttons within the footer are visible only when the **Card.AllowCommonActions** property is `true`. The **PrimaryActionName** and **SecondaryActionName** properties define button names. 
```

File: README.md [hunk 7]
```diff
@@ -0,0 +1,279 @@
...
+            protected void NotifyPropertyChanged([CallerMemberName] string propertyName = "") {
+                PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
+            }
+        }
+    }
+    ```
+
+## C. Footer (Optional)
+
+The footer area includes buttons that hide or apply all the items from this group.
```
Comment (Abadzhev) [inline README.md:178]:
```suggestion
The footer area includes buttons that can initiate list-level actions (such as "Hide All"). 
```

File: README.md [hunk 8]
```diff
@@ -0,0 +1,279 @@
...
+            PreviewItemsCount = 3,
+        });
+        // ...
+    }
+    // ...
+}
+```
+
+
+Follow the steps below to implement commands that performs operations over all **CollectionView** items within the card:
```
Comment (Abadzhev) [inline README.md:214]:
```suggestion
Follow the steps below to implement commands that process all **CollectionView** items within the card:
```

File: README.md [hunk 9]
```diff
@@ -0,0 +1,279 @@
...
+
+1. Specify the [SimpleButton.Command](https://docs.devexpress.com/MAUI/DevExpress.Maui.Controls.SimpleButton.Command) property to define the click commands for both footer buttons. These commands runs only when the
```
Comment (Abadzhev) [inline README.md:216]: This is clearly incomplete - please update and I'll review this part then.

File: README.md [hunk 10]
```diff
@@ -0,0 +1,279 @@
...
+            <DataTrigger TargetType="HorizontalStackLayout" Binding="{Binding AllowCommonActions}" Value="False">
+                <Setter Property="IsVisible" Value="False"/>
+            </DataTrigger>
+        </HorizontalStackLayout.Triggers>
+        <dxco:SimpleButton Text="{Binding PrimaryActionName}" Command="{Binding SecondaryActionCommand}" .../>
+        <dxco:SimpleButton Text="{Binding SecondaryActionName}" Command="{Binding PrimaryActionCommand}" .../>
+    </HorizontalStackLayout>
+    ```
+
+1. Define the **Card** class's **PrimaryAction** and **SecondaryAction** commands:
```
Comment (Abadzhev) [inline README.md:230]:
```suggestion
1. Define the **Card.PrimaryAction** and **Card.SecondaryAction** commands:
```

File: README.md [hunk 11]
```diff
@@ -0,0 +1,279 @@
...
+<img width="50%" src="https://user-images.githubusercontent.com/12169834/223110944-4904bf34-da91-4685-9656-fb7e09905d42.png"/>
+
+## Used Controls and Their Properties
+
+**A. Header**
+
+  * [Label](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/label?view=net-maui-7.0): [Text](https://learn.microsoft.com/en-us/dotnet/api/microsoft.maui.controls.label.text?view=net-maui-7.0)
+  * [SimpleButton](https://docs.devexpress.com/MAUI/DevExpress.Maui.Controls.SimpleButton): [Command](https://docs.devexpress.com/MAUI/DevExpress.Maui.Controls.SimpleButton.Command)
+
+**B. List Items**
```
[the remark this answers is not in the document]
Comment (friedcucumber) [reply README.md:14]: Added, thanks

File: README.md [hunk 12]
```diff
@@ -0,0 +1,280 @@
...
+
+    The **NavigateToAll** method calls the [GoToAsync](https://learn.microsoft.com/en-us/dotnet/api/microsoft.maui.controls.shell.gotoasync?view=net-maui-7.0#microsoft-maui-controls-shell-gotoasync(microsoft-maui-controls-shellnavigationstate-system-collections-generic-idictionary((system-string-system-object)))) method to open the detail view.
+
+3. Specify the [QueryProperty](https://learn.microsoft.com/en-us/dotnet/api/microsoft.maui.controls.querypropertyattribute?view=net-maui-7.0) attribute for the **CompleteListViewModel** class to pass the clicked header context to the command registered in the previous step:
+   
+   ```csharp
+   namespace CollectionViewWithActionButtons.ViewModels {
+       [QueryProperty(nameof(ParentCard), "Parent")]
+       internal class CompleteListViewModel : INotifyPropertyChanged {
+           public Card parentCard;
```
Comment (friedcucumber) [inline README.md:88]:
```suggestion
           private Card parentCard;
```

File: README.md (hunk 3 above)
Comment (friedcucumber) [reply README.md:89]: Fixed, thanks

File: README.md [hunk 13]
```diff
@@ -0,0 +1,280 @@
+# List Cards
+
+This example shows you how to implement a List Card. This UI element allows you to preview a few items from a lengthy list, access the complete list, and execute custom actions.
```
Comment (arminal) [inline README.md:3]:
```suggestion
This example shows you how to implement a List Card - a UI element that allows you to preview several items from a lengthy list, access the complete list, and execute custom actions. 
```

**Pull request closed without merging**